lambda_functions/user_flows/conftest.py

```diff
@@ -9,3 +9,21 @@ def pytest_generate_tests(metafunc):
     option_value = metafunc.config.option.environment
     if 'environment' in metafunc.fixturenames and option_value is not None:
         metafunc.parametrize("environment", [option_value],  scope="session")
+
+
+def pytest_runtest_logreport(report):
+    if report.longrepr is None:
+        return
+    for tb_repr, *_ in report.longrepr.chain:
+        for entry in tb_repr.reprentries:
+            if entry.reprfuncargs is not None:
+                args = entry.reprfuncargs.args
+                for idx, (name, value) in enumerate(args):
+                    if "service_test_api_key" in name:
+                        args[idx] = (name, "********")
+            if entry.reprlocals is not None:
+                lines = entry.reprlocals.lines
+                for idx, line in enumerate(lines):
+                    if line.startswith("service_test_api_key"):
+                        lines[idx] = "service_test_api_key          = '*********'"
+
```

Comment:
This works for `service_test_api_key`, but there is another variable in use by the tests, which if those test fail, could then print out that variable: `service_api_key`. So this code need to look for both variables.